source/extensions/stat_sinks/hystrix/hystrix.h

```diff
@@ -0,0 +1,146 @@
+#pragma once
+
+#include <map>
+#include <memory>
+#include <vector>
+
+#include "envoy/server/admin.h"
+#include "envoy/server/instance.h"
+#include "envoy/stats/stats.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace StatSinks {
+namespace Hystrix {
+
+typedef std::vector<uint64_t> RollingWindow;
+typedef std::map<const std::string, RollingWindow> RollingStatsMap;
+
+struct ClusterStatsCache {
```

Comment:
I  think this should be a class not a struct.

https://google.github.io/styleguide/cppguide.html#Structs_vs._Classes

But if this is a clear pattern in Envoy it's OK to leave it as a struct, in which case you don't need to suffix the member vars with _

Reply:
Actually by that definition, it seems to me that it should be a struct and not a class. It passively holds the rolling windows, which are manipulated by the sink's methods. What do you think?

Reply:
OK, that's fine; I guess the methods are just printers. And actually the _ suffix is called out by STYLE.md as a local rule.  Leave as is then :)